tempesta_fw/work_queue.h

```diff
@@ -51,10 +51,10 @@ int __tfw_wq_push(TfwRBQueue *wq, void *ptr, bool sync);
 int tfw_wq_pop(TfwRBQueue *wq, void *buf);
 
 static inline int
-tfw_wq_push(TfwRBQueue *wq, void *ptr, int cpu, struct irq_work *work,
+tfw_wq_push(TfwRBQueue *q, void *ptr, int cpu, struct irq_work *work,
```

Comment:
I just touched `tfw_wq_push()` at some place and made the change for name consistency. However I didn't spend time for full names cleanup - we can always do it background. Generally shorter names are preffered if it's still clear what do they mean.